app/src/main/java/com/hammer/talkbbokki/presentation/navigation/TalkbbokkiNavHost.kt

```diff
@@ -74,7 +75,10 @@ fun TalkbbokkiNavHost(
         )
         bookmarkGraph(
             navigateToDetail = { item ->
-                navController.navigate(DetailDestination.route + "/${item.id}")
+                navController.navigate(
+                    DetailDestination.route +
+                        "?level=${item.category.uppercase(Locale.getDefault())}&id=${item.id}&tag=${item.tag}&topic=${item.name}&shareLink=${item.shareLink}"
```

Comment:
🚫 **[ktlint]** <max-line-length> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Exceeded max line length (100) (cannot be auto-corrected)